features/upgrade/README.md

```diff
@@ -1,3 +1,9 @@
+# Upgrading from rspec-rails 4.x to version 5
+
+RSpec Rails 5 is a major version under semantic versioning, it also follows our new versioning strategy for RSpec-Rails, which is to keep in step with Rails supported versions. Thus it supports 5.2, 6.0 and 6.1. There are no changes require to upgrade to RSpec Rails 5 if you are using a supported version of Rails.
```

Comment:
"require" -> "required"

I really need to learn how to use GitHub's suggestion feature.